src/css/profile/mobile/common/popup.less

```diff
@@ -93,6 +88,144 @@
 	}
 }
 
+@media (min-width: 0px) and (max-width: 479px) {
+	@media (min-height: 600px) and (max-height: 959px) {
+		.ui-popup {
+			width: 100%;
+			margin: 0 auto;
+			background-color: rgba(255, 255, 255, 0);
+			margin-top: 40px;
+			margin-bottom: 16px;
+			padding-top: 16px;
+			.ui-popup-header {
+				font-size: 20px;
```

Comment:
How about the font size define in the theme.font.less?

//**************************************************************************
//                   Popup
//**************************************************************************
@font_size_popup_title_text: 50 * @unit_base;		// T121: Popup title text
@font_size_popup_title_over_sub_text: 40 * @unit_base;	// T1211: Popup title with subtitle
@font_size_popup_title_sub_text: 28 * @unit_base;	// T1212: Popup subtitle

Reply:
I fixed.

done.